tests/unit/controllers/show-project-test.js

```diff
@@ -27,4 +26,39 @@ module('Unit | Controller | show-project', function(hooks) {
       fitBounds() {},
     });
   });
+
+  test('hasBBLFeatureCollectionGeometry is null when geometry is null', function(assert) {
+    const controller = this.owner.lookup('controller:show-project');
+    assert.ok(controller);
+
+
+    controller.model = {
+      bbl_featurecollection: {
+        features: [{
+          type: 'Feature',
+          geometry: null,
+        }],
+      },
+    };
```

Comment:
I woulda preferred using mirage to set a real model on the controller, but the stub should do for now.

Reply:
Okay I'll try to set one up

Reply:
Changed!